### PR TITLE
Wikipedia lookup

### PR DESCRIPTION
**Pull request by andreas-grafberger**
Added wikipedia lookup

### CODE REVIEW

File: intents/WikipediaIntent.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            sum = "Could not find page for " + topicName
+        except DisambiguationError as e:
+            options = e.options
+            if len(options) == 0:
+                sum = "Could not find page for " + topicName
+            else:
+                firstOption = options[0]
+                wikiPage = page(firstOption)
+                sum = wikiPage.summary
+        else:
```
Comment (sarahsporck) [inline intents/WikipediaIntent.py:33]: What´s that?

File: intents/WikipediaIntent.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    def execute(str, bot, chat_id):
+        answer = ""
+        keyword = WikipediaIntent.filterKeyword(str)
+        if keyword is None:
+            answer = "I don't know what to do, sorry. Please reword your request."
+        else:
+            answer = WikipediaIntent.getUrlSummaryForTopic(keyword)
+
+        bot.send_message(chat_id, answer)
+        return ""
```
Comment (sarahsporck) [inline intents/WikipediaIntent.py]: Why do we keep `return ""`?

Comment (andreas-grafberger) [reply intents/WikipediaIntent.py]: Good question! Will be removed.

File: intents/WikipediaIntent.py (hunk 1 above)
Comment (andreas-grafberger) [reply intents/WikipediaIntent.py:33]: If no exception is thrown

**Review by sarahsporck — APPROVED** [review]

**Review by andre111 — APPROVED** [review]
Looks goood

**Pull request merged**